m3ntorship-Posts-1.0.0-swagger.json

```diff
@@ -591,6 +623,19 @@
             "type": "string"
           }
         }
+      },
+      "Report": {
+        "type": "object",
+        "properties": {
+          "postId": {
```

Comment:
Generalize property name, to be able to use it for other things in the future without the need to change/refactor.
Name it `id` instead of `postId` for example.

Reply:
And add another property to specify the type of report.
For example, `"type":"post"`. which is our current case.